src/main/java/org/junitpioneer/vintage/ExpectedExceptionExtension.java

```diff
@@ -44,14 +44,14 @@ public void handleTestExecutionException(ExtensionContext context, Throwable thr
 				.filter(expected -> expected.isInstance(throwable))
 				.isPresent();
 
-		// in the `afterTestExecution` callback we have to pass or fail the test
-		// depending on whether the exception was thrown or not;
-		// to do that we need to register whether the exception was thrown;
```

Comment:
@nicolaiparlog You were wondering why this was changes and you thought it was because the lower case at the start of the line. No it's not. The `;` are the violation as sonarcube thinks it's outcommented code, see [here](https://sonarcloud.io/project/issues?id=junit-pioneer_junit-pioneer&open=AXDuy1qtYLFyXCsBD7Mm&resolved=false&severities=MAJOR#)